example/minio_standalone/main.tf

```
@@ -22,7 +22,8 @@ module "minio" {
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
   use_canary                      = true
-
+  cpu_proxy                       = 200
+  memory_proxy                    = 128
```

I think it should be
```suggestion
resource_proxy = {
  cpu = 200
  memory = 128
}
```